seed/data_importer/views.py

```diff
@@ -316,17 +316,21 @@ def create_from_pm_import(self, request):
 
         # This list should cover the core keys coming from PM, ensuring that they map easily
         # We will also look for keys not in this list and just map them to themselves
-        pm_key_to_column_heading_map = {
-            u'address_1': u'Address',
-            u'city': u'City',
-            u'state_province': u'State',
-            u'postal_code': u'Zip',
-            u'county': u'County',
-            u'country': u'Country',
-            u'property_name': u'Property Name',
-            u'property_id': u'Property ID',
-            u'year_built': u'Year Built',
-        }
+        # pm_key_to_column_heading_map = {
+        #     u'address_1': u'Address',
+        #     u'city': u'City',
+        #     u'state_province': u'State',
+        #     u'postal_code': u'Zip',
+        #     u'county': u'County',
+        #     u'country': u'Country',
+        #     u'property_name': u'Property Name',
+        #     u'property_id': u'Property ID',
+        #     u'year_built': u'Year Built',
+        # }
+        # so now it looks like we *don't* need to override these, but instead we should leave all the headers as-is
```

Comment:
this makes sense since after downloading and importing the user is dropped at the mapping screen.